scripts/build_jacocorunner/build_jacocorunner_bazel_5.0+.sh

```diff
@@ -36,6 +36,10 @@
 
 set -e
 
+# Ensure Java 8 is used for building Jacoco (experienced issue when using e.g. Java 17).
+# You may need to change this on your system.
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+export PATH=$JAVA_HOME/bin:$PATH
```

Comment:
I think these settings are too specific (I have different place for JAVA_HOME, and I can switch java versions using jenv in my local environment).

What do you think to add a sentence "Ensure Java 8 is used for building Jacoco" in `docs/coverage.md` and remove these `export`s?

Reply:
Whatever solution works for you. Should I comment these lines and leave a fixme?

Reply:
Updated my commit. Now, you need to manually ensure, and the script will error out if the wrong Java version is set.